woocommerce/woocommerce-ios
# Order Details > Edit Order Status: the current order status should be preselected in the list

Fixes #1368 

Update release notes:


## Changes
- Passed optional `OrderStatus` to `OrderStatusListViewController`
- In `OrderStatusListViewController`, preselect the status if possible in `viewDidLoad`
- Because `setHighlighted` (and `setSelected`) is called every time a cell is initialized, also set the cell's accessory type to `checkmark` if the cell's selected state is true in `StatusListTableViewCell`

## Testing
- Go to the Orders tab
- Tap on an Order
- Edit the Order status by tapping the pencil button by the status --> the current status should be pre-selected and the "Apply" button should be disabled
- Try tapping on other statuses --> there should only be one status selected
- Tap on a different order status from the Order's current status and tap "Apply" --> the Order status should be updated
